### ISSUE TITLE
i can't find the wave,when i try the deepvoice3_single_speaker_tts_en_demo

### ISSUE THREAD

**Issue by Yaque** [r9y9/deepvoice3_pytorch#102]
`python3 deepvoice3_single_speaker_tts_en_demo.py 
True
[nltk_data] Downloading package cmudict to /home/yaque/nltk_data...
[nltk_data]   Package cmudict is already up-to-date!

True
True
Load checkpoint from: 20180505_deepvoice3_checkpoint_step000640000.pth
0 This is a pig
/home/yaque/.local/lib/python3.5/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
<IPython.lib.display.Audio object>
1 There's a way to measure the acute emotional intelligence that has never gone out of style.
<IPython.lib.display.Audio object>
2 President Trump met with other leaders at the Group of 20 conference.
<IPython.lib.display.Audio object>
3 The Senate's bill to repeal and replace the Affordable Care Act is now imperiled.
<IPython.lib.display.Audio object>
4 Generative adversarial network or variational auto-encoder.
<IPython.lib.display.Audio object>
5 The buses aren't the problem, they actually provide a solution.
<IPython.lib.display.Audio object>
6 peter piper picked a peck of pickled peppers how many peppers did peter piper pick.
<IPython.lib.display.Audio object>
7 Some have accepted this as a miracle without any physical explanation.
<IPython.lib.display.Audio object>
<IPython.lib.display.Audio object>
`
i change somg thing and run deepvoice3_single_speaker_tts_en_demo in my computer.
but i can't find the wave file.

**Comment by Yaque** [comment]
now i  think i kwon why

**Comment by Yaque** [comment]
ok ,just i don't save the wav. and i don't kown the IPython's function.
